### DISCUSSION TITLE
Notify error

### DISCUSSION THREAD

**Discussion by iamshm** [LazyVim/LazyVim#2864] [Q&A] [answered]
<img width="1501" alt="Screenshot 2024-03-28 at 7 33 45 PM" src="https://github.com/LazyVim/LazyVim/assets/43462941/73a5db5f-b34e-4e2b-9154-05308a169e1b">

every time I press leader K 
this notify things pops up even when there is information

Can anyone help me out please?

**Comment by 0x2E** [comment, marked as answer]
```lua
  {
    "folke/noice.nvim",
    opts = function(_, opts)
      table.insert(opts.routes, {
        filter = {
          event = "notify",
          find = "No information available",
        },
        opts = { skip = true },
      })
    end,
  },
```

ref: https://youtu.be/fFHlfbKVi30?list=LL&t=2355

**Reply by iamshm** [reply]
This worked. Thanks mate!
